spec/javascript/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -137,9 +137,10 @@ describe('document-capture/components/document-capture', () => {
     submitButton = getByText('forms.buttons.submit.default');
     expect(isFormValid(submitButton.closest('form'))).to.be.true();
 
+    // eslint-disable-next-line require-await
```

Comment:
This change is somewhat strange and unexpected. Without it, it identifies the `new Promise` argument function as being `async` without and `await` call, but it's not an `async` function, and I'm not sure why it would suddenly start being flagged.

My guess is that some transitive dependency of what's changed in `yarn.lock` changed how the file is being parsed?